benchmarks/libmagic/README.md

```diff
@@ -0,0 +1,236 @@
+# Benchmarking libmagic with minimal magic files
+
+To assess the performance improvement gained by using a minimal magic file, we created the benchmark
+`main.c`. This benchmark measures the time needed to find the MIME type of a file with
+a given compiled magic file.
```

Comment:
Mention here already the 14.45% gain between full mime db and mime db covering only X mimetypes.

Reply:
Can you also update the main README.md. 
* Add a link to this section
* Mention the performance improvements of 14.45% in the first paragraph (as it basically shows the value of this project).

Reply:
Do you have a hypothesis why we see an improvement of 14.45% here but a 30% performance improvement for overall squid performance?

Reply:
Maybe it's worth to mention in the main README.md or by adding one to `/benchmars/README.md` what `/e2e` vs `/libmagic`vs `/mod` is about.

Reply:
> Do you have a hypothesis why we see an improvement of 14.45% here but a 30% performance improvement for overall squid performance?

If you look at the results, they vary a lot, it would probably be better to do some kind of average difference rather than just one file.